src/twisted/conch/scripts/ckeygen.py

```diff
@@ -150,7 +150,7 @@ def generateECDSAkey(options):
 
     if not options['bits']:
         options['bits'] = 256
-    # OpenSSH supports only nistp curves.
+    # OpenSSH supports only mandatory sections of RFC5656.
```

Comment:
would it be better to say openssh doesn't support recommended curves described in section 10.2 of RFC5656. https://tools.ietf.org/html/rfc5656#section-10.2?